.golangci.yml

```diff
@@ -59,31 +59,28 @@ linters-settings:
 
 linters:
   enable:
-    - deadcode
     - errcheck
     - goconst
     - revive
     - ineffassign
     - misspell
-    - structcheck
     - unconvert
-    - varcheck
     - govet
     - typecheck
     - depguard
     - exportloopref
-
-  # TODO(dannyk): restore these and fix linter errors in separate PR
-  disable:
-    - unused
     - gofmt
     - goimports
     - gosimple
     - staticcheck
+  disable:
+    - unused
     - unparam
 
 issues:
   exclude:
     - Error return value of .*log\.Logger\)\.Log\x60 is not checked
     - Error return value of .*.Log.* is not checked
     - Error return value of `` is not checked
+
+  fix: true
```

Comment:
This automatically fixes lint errors that can be fixed automatically.
This will only work when running `make lint` from your own workstation.